docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ghcr.io/cybozu/ubuntu:22.04 as stage1
+FROM ghcr.io/cybozu/ubuntu:24.04 AS stage1
```

Comment:
It would be good to build binaries on Ubuntu 24.04.
* https://github.com/cybozu-go/setup-hw/blob/v1.17.1/.github/workflows/main.yaml
* https://github.com/cybozu-go/setup-hw/blob/v1.17.1/.github/workflows/release.yaml